modules/reference/pages/spacecmd/whoamitalkingto.adoc

```diff
@@ -0,0 +1,12 @@
+[[ref-spacecmd-whoamitalkingto]]
```

Comment:
How is this related to removing underscores?

Reply:
This was in redundant file that was now removed.